packages/expo-medplum-polyfills/src/ExpoMedplumPolyfillsModule.ts

```diff
@@ -0,0 +1,103 @@
+import type { MedplumClient } from '@medplum/core';
+import { decode, encode } from 'base-64';
+import { CryptoDigestAlgorithm, digest } from 'expo-crypto';
+import expoWebCrypto from 'expo-standard-web-crypto';
+import 'react-native-url-polyfill/auto';
+import { TextEncoder } from 'text-encoding';
+
+type ExtendedExpoCrypto = typeof expoWebCrypto & {
+  subtle: {
+    digest: (algorithm: AlgorithmIdentifier, data: BufferSource) => Promise<ArrayBuffer>;
+  };
+};
+
+export function polyfillMedplumWebAPIs(medplum: MedplumClient): void {
+  if (typeof window.crypto === 'undefined') {
+    Object.assign(expoWebCrypto, {
+      subtle: { digest: polyfilledDigest },
+    }) satisfies ExtendedExpoCrypto;
+
+    Object.defineProperty(window, 'crypto', {
+      configurable: true,
+      enumerable: true,
+      get: () => expoWebCrypto,
+    });
+  }
+
+  if (typeof window.location === 'undefined') {
+    const locationUrl = new URL('/', medplum.getBaseUrl()) as URL & { assign: () => void };
```

Comment:
`getBaseUrl` returns the API base URL, not the current location.  

I'm not entirely sure what the default behavior of `window.location` should be.... probably something super generic such as `http://localhost:80`?